### ISSUE TITLE
Isolate tests for Dockerfile builds using Poetry build includes/excludes

### ISSUE THREAD

**Issue by d33bs** [cytomining/pycytominer#397]
> Here's another possible change at your discretion. This would make it a bit easier with the context management.
>
> 1. Move `./build/docker/Dockerfile` to `./Dockerfile`
> 2. Add the `tests` directory as an include for the sdist build artifact but not wheel ([instructions](https://python-poetry.org/docs/pyproject/#include-and-exclude))
>
> I think the end result would be that the commands `docker build . --target testing` would work both in a cloned repo root or in an extracted sdist.

_Originally posted by @kenibrewer in https://github.com/cytomining/pycytominer/issues/395#issuecomment-2034498055_

**Comment by axiomcura** [comment]
Hey @d33bs, given the migration to uv in #650, does this issue still apply?

**Comment by d33bs** [comment]
Yes, I think this still applies, but is low priority generally. We could just say something like "build manager" instead of Poetry if we want to be less specific (in case that's why you asked when searching / commenting on this).
